pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -673,36 +921,59 @@ func TestDryRunPreemption(t *testing.T) {
 				t.Fatal(err)
 			}
 
-			state := framework.NewCycleState()
-			// Some tests rely on PreFilter plugin to compute its CycleState.
-			if status := fwk.RunPreFilterPlugins(context.Background(), state, tt.pod); !status.IsSuccess() {
-				t.Errorf("Unexpected PreFilter Status: %v", status)
-			}
-
 			nodeInfos, err := snapshot.NodeInfos().List()
 			if err != nil {
 				t.Fatal(err)
 			}
-			got := dryRunPreemption(context.Background(), fwk.PreemptHandle(), state, tt.pod, nodeInfos, tt.pdbs)
-			if err != nil {
-				t.Fatal(err)
-			}
-			// Sort the values (inner victims) and the candidate itself (by its NominatedNodeName).
-			for i := range got {
-				victims := got[i].Victims().Pods
-				sort.Slice(victims, func(i, j int) bool {
-					return victims[i].Name < victims[j].Name
+			sort.Slice(nodeInfos, func(i, j int) bool {
+				return nodeInfos[i].Node().Name < nodeInfos[j].Node().Name
+			})
```

Comment:
Is this necessary? I'd assume the test cases to have specified the nodes ordered by name.

Reply:
Yes, otherwise the returned node list is in arbitrary order for some reason and there's no way to deterministically test for correct offsets and wrap arounds.